low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -20,72 +20,92 @@
-                                           sspl_config_path,
-                                           PRODUCT_BASE_DIR,
-                                           GLOBAL_CONFIG_INDEX,
-                                           SSPL_CONFIG_INDEX,
-                                           CONFIG_SPEC_TYPE,
-                                           enabled_products)
+from cortx.utils.validator.v_service import ServiceV
+from files.opt.seagate.sspl.setup.setup_error import SetupError
+from framework.base import sspl_constants as consts
+from framework.utils.utility import Utility
 
 
 class SSPLPostInstall:
     """Prepare environment for SSPL service."""
 
-    name = "SSPL Post Install"
-
-    def __init__(self, args: str):
-        """Ssplpostinstall init."""
-        self.global_config_url = args.config[0]
-        self.name = "sspl_post_install"
-        self._script_dir = os.path.dirname(os.path.abspath(__file__))
-        self.RSYSLOG_CONF = "/etc/rsyslog.d/0-iemfwd.conf"
-        self.RSYSLOG_SSPL_CONF = "/etc/rsyslog.d/1-ssplfwd.conf"
-        self.PACEMAKER_INSTALLATION_PATH = "/lib/ocf/resource.d/seagate/"
-        self.ENVIRONMENT = "PROD"
-
-        # Load and dump provsioner supplied global config in required format
-        self.prvsnr_global_config = "prvsnr_global_config"
-        global_config_copy_loc = "/etc/sspl_global_config_copy.%s" % CONFIG_SPEC_TYPE
-        self.global_config_copy_url = "%s://%s" % (CONFIG_SPEC_TYPE,
-                                                   global_config_copy_loc)
-        Conf.load(self.prvsnr_global_config, self.global_config_url)
-        self.dump_global_config()
-
-    def dump_global_config(self):
-        """Dump provisioner global config and load it."""
-        url_spec = urlparse(self.global_config_copy_url)
-        path = url_spec.path
-        store_loc = os.path.dirname(path)
-        if not os.path.exists(store_loc):
-            os.makedirs(store_loc)
-        with open(path, "w") as f:
-            f.write("")
-        # Make copy of global config
-        Conf.load(GLOBAL_CONFIG_INDEX, self.global_config_copy_url)
-        Conf.copy(self.prvsnr_global_config, GLOBAL_CONFIG_INDEX)
-        Conf.save(GLOBAL_CONFIG_INDEX)
+    name = "sspl_post_install"
+
+    def __init__(self):
+        """Initialize varibales for post install."""
+        self.user = "sspl-ll"
+        self.sspl_log_path = "/var/log/%s/sspl/" % consts.PRODUCT_FAMILY
+        self.sspl_bundle_path = "/var/%s/sspl/bundle/" % consts.PRODUCT_FAMILY
+        self.state_file = "%s/state.txt" % consts.DATA_PATH
+        self.dbus_service = DbusServiceHandler()
 
     def validate(self):
         """Check below requirements are met in setup.
 
-        1. required python 3rd party packages are installed
-        2. required rpm dependencies are installed
-        3. product specified in global config is supported by SSPL
+        1. Given product is supported by SSPL
```

Comment:
Check if given product ....
same for setup

Reply:
Done.